_posts/2021-05-14-Spring-Boot-TestConfiguration.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Spring Boot @TestConfiguration Annotation
+categories: [spring-boot]
+date: 2021-05-15 00:00:00 +0530
+modified: 2021-05-15 06:00:00 +1000
+excerpt: "Create custom beans or override specific bean behaviors during unit testing of Spring Boot applications "
+image:
+ auto: 0035-switchboard
+---
+In this article, we will see how to create custom beans or override specific beans during unit testing by using the `@TestConfiguration` annotation.
```

Comment:
Add one or two lines on benefits or killer features that will get the reader interested to read further.

Reply:
Do you think the Section `Configuration vs TestConfiguration` could be moved here?